DFE-Digital/technical-guidance
# Updated code to create DNS Zone and azure front door

## Description
Deploys DNS Zone and front door for technical guidance on AKS.

## How to review
- Check the DNS zones created for technical-guidance.education.gov.uk
- Check azure front door profile s189p01-techg-dom-fd created.
